vscode/microsoft-kiota/src/extension.ts

```diff
@@ -237,7 +240,7 @@ export async function activate(
             }
           }
 
-          deepLinkParams = {};  // Clear the state after the generation
+          setDeepLinkParams({});  // Clear the state after the generation
```

Comment:
Does this really clear the state? From the function definition, I can see that doing this won't change the deepLink Params in any way.

Reply:
That's a good callout @SilasKenneth. I've created an explicit function for clearing the state